trillian/integration/ct_functions.sh

```diff
@@ -21,6 +22,7 @@ CT_COMBINED_CFG=
 # If etcd and Prometheus are configured, it also populates:
 #  - ETCDISCOVER_PID : pid of etcd service watcher
 #  - PROMETHEUS_PID  : pid of local Prometheus server
+#  - PROMETHEUS_CFGDIR  : Prometheus configuration directory
```

Comment:
nit: were these meant to line up horizontally?

Reply:
Fixed.